src/controllers.php

```diff
@@ -26,33 +26,24 @@
 
 // Specific event
 $app->get('/event/{id}', function ($id) use ($app) {
-    $event = $app['meetup']->getEvent($id);
 
     /** @var RandomService $randomService */
     $randomService = $app['random'];
 
+    $event = $app['meetup']->getEvent($id);
     $client = new Client();
     $checkins = array_filter($client->lrange('checkin_'.$id, 0, 300));
 
-    $winners = (count($checkins) > 0)? $randomService->getRandomNumbers(count($checkins)) : array();
-    return $app['twig']->render(
-        'event.html.twig',
-        array('event' => $event, 'winners' => $winners, 'checkins' => $checkins)
-    );
-})->bind('event');
+    $checkinsNumber = count($checkins);
 
-// Check-in page for Event
-$app->get('/event/{id}/checkin', function ($id, Request $request) use ($app) {
-
-    $event = $app['meetup']->getEvent($id);
-    $client = new Client();
-    $checkins = array_filter($client->lrange('checkin_'.$id, 0, 300));
+    $winners = ($checkinsNumber) ? $randomService->getRandomNumbers($checkinsNumber) : [];
```

Comment:
I checked the service and there's no harm on calling it with `0` so `$winners = $randomService->getRandomNumbers(count($checkins));` would be more readable IMO.